swan-lake/resources/featured-scenarios/write-a-restful-api-with-ballerina.md

```diff
@@ -28,76 +26,161 @@ This guide will walk you through creating a RESTful API with two endpoints. In t
+    >**Tip:** You need to complete the generated record by adding the pipe signs to mark the record as a closed one and adding the `readonly` descriptor to the `iso_code` field to make it non-modifiable as it is the key of the table, which cannot be represented in the JSON format.
+
+    ```json
+    {
+        "iso_code": "AFG",
+        "country": "Afghanistan",
+        "cases": 159303,
+        "deaths": 7386,
+        "recovered": 146084,
+        "active": 5833
+    }
+    ```
+    ![Create data record](/learn/images/featured-scenarios/write-a-restful-api-with-ballerina/create-data-record.gif)
+
+2. Create the table, as shown below.
+
+    >**Tip:** Enter `CovidEntry` as the table type, `iso_code` as the key, and `covidTable` as the variable type when creating the table.
+
+    ![Create data table](/learn/images/featured-scenarios/write-a-restful-api-with-ballerina/create-data-table.gif)
+
+3. Replace the `{key: value}` placeholder of the generated table with the following entries to add the initial data to the table.
+
+    ```
+        {
+            iso_code: "AFG", 
+            country: "Afghanistan", 
+            cases: 159303, 
+            deaths: 7386, 
+            recovered: 146084, 
+            active: 5833
+        },
+        {
+            iso_code: "SL", 
+            country: "Sri Lanka", 
+            cases: 598536, 
+            deaths: 15243, 
+            recovered: 568637, 
+            active: 14656
+        },
+        {
+            iso_code: "US", 
+            country: "USA", 
+            cases: 69808350, 
+            deaths: 880976, 
+            recovered: 43892277, 
+            active: 25035097
+        }
+    ```
+The generated record and the table will be as follows.
 
 ```ballerina
-public type CovidEntry record {|
+type CovidEntry record {|
     readonly string iso_code;
     string country;
-    decimal cases;
-    decimal deaths;
-    decimal recovered;
-    decimal active;
+    int cases;
+    int deaths;
+    int recovered;
+    int active;
 |};
 
-public final table<CovidEntry> key(iso_code) covidTable = table [
-    {iso_code: "AFG", country: "Afghanistan", cases: 159303, deaths: 7386, recovered: 146084, active: 5833},
-    {iso_code: "SL", country: "Sri Lanka", cases: 598536, deaths: 15243, recovered: 568637, active: 14656},
-    {iso_code: "US", country: "USA", cases: 69808350, deaths: 880976, recovered: 43892277, active: 25035097}
+final table<CovidEntry> key(iso_code) covidTable = table [
+    {
+        iso_code: "AFG",
+        country: "Afghanistan",
+        cases: 159303,
+        deaths: 7386,
+        recovered: 146084,
+        active: 5833
+    },
+    {
+        iso_code: "SL",
+        country: "Sri Lanka",
+        cases: 598536,
+        deaths: 15243,
+        recovered: 568637,
+        active: 14656
+    },
+    {
+        iso_code: "US",
+        country: "USA",
+        cases: 69808350,
+        deaths: 880976,
+        recovered: 43892277,
+        active: 25035097
+    }
```

Comment:
Need to update this also along with https://github.com/ballerina-platform/ballerina-dev-website/pull/8664/files#r1496589026

Reply:
We will have to update the gifs also. Anyway we have to redo based on https://github.com/ballerina-platform/ballerina-dev-website/pull/8664/files#r1496577265